src/Core/Core.js

```diff
@@ -338,7 +338,7 @@ export default class Core {
       if (serializedComponents[0].variants.uniqueVariants) {
         let docClass =
           this.componentInfoObjects.allComponentClasses[
-            serializedComponents[0].componentType
+          serializedComponents[0].componentType
```

Comment:
We need to figure out why we are getting different prettier results. When I save this document, it goes back to the original spacing!

Reply:
Just ran prettier on my machine and saved it to resolve this for now.